src/types.ts

```diff
@@ -239,10 +246,12 @@ export interface ICrawler {
   jsonPointer?: string;
   pointerGraph: DepGraph<string>;
   pointerStemGraph: DepGraph<string>;
-  computeGraph: (target: any, parentPath: string[], parentPointer: string, pointerStack: string[]) => void;
+  resolvers: Array<Promise<IUriResult>>;
+  computeGraph: (target: any, parentPath: string[], parentPointer: string, pointerStack?: string[]) => void;
 }
 
 export interface ICrawlerResult {
-  result: any;
-  errors: IResolveError[];
+  jsonPointer?: string;
```

Comment:
Hmm, what's driving these changes?

Reply:
Nothing, ICrawlerResult didn’t seem to used anywhere anyways. Meant to revert this.